tests/test_imageCollection_func.py

```diff
@@ -68,3 +68,9 @@ def test_image_collection_group_by_query():
     response = SQL2GEE(JsonSql(sql).to_json(), geojson=r).response()
     assert len(response) == 10, "BASIC limit query incorrect"
     return
+
+def test_image_collection_group_by_with_geometry_where_condition():
+    sql = "select system:index, first(q25) as q25,first(q50) as q50, first(q75) as q75, mean(q75) as q75_mean from 'projects/resource-watch-gee/cli_051_nexgddp_annual_tasmin' where ST_INTERSECTS(ST_SetSRID(ST_GeomFromGeoJSON('{\"type\":\"Point\",\"coordinates\":[-110.22939224192194,19.986126139624318]}'),4326),the_geom) group by system:index"
+    response = SQL2GEE(JsonSql(sql).to_json()).response()
+    assert len(response) > 0, f"Empty response: {response}"
```

Comment:
Testing that a response is not empty does not validate that the query is returning what it should, it just ensures it's not returning one of the things that it shouldn't. Please make these tests more restrictive, so we can have stronger confidence in the results.